notable-changes/1.22.1/notable-change.md

```diff
@@ -0,0 +1,86 @@
+<!--{"metadata":{"requirement":"RECOMMENDED","type":"INTERNAL","category":"CONFIGURATION","additionalFiles":0}}-->
+
+# Updating Kyma Environment Broker: Zones Discovery
+
+> [!NOTE]
+> This is a recommended change. To enable Zones Discovery for AWS, update the Kyma Environment Broker (KEB) configuration.
+
+## Prerequisites
+
+- KEB is configured to use the AWS provider.
+
+## What's Changed
+
+With the [Zones Discovery](https://github.com/kyma-project/kyma-environment-broker/blob/main/docs/contributor/03-55-zones-discovery.md) feature, KEB can dynamically retrieve available zones from the hyperscaler (currently only AWS) instead of relying on statically configured zones in the `providersConfiguration`.
```

Comment:
```suggestion
With the [Zones Discovery](https://github.com/kyma-project/kyma-environment-broker/blob/main/docs/contributor/03-55-zones-discovery.md) feature, KEB can dynamically retrieve available zones from the hyperscaler (currently only AWS) instead of relying on statically configured zones in `providersConfiguration`.
```